opsdroid/__main__.py

```diff
@@ -68,7 +68,7 @@ def print_example_config(ctx, param, value):
 def edit_files(ctx, param, value):
     """Open config/log file with favourite editor."""
     if value == "config":
-        file = DEFAULT_CONFIG_PATH
+        file = f'"{DEFAULT_CONFIG_PATH}"'
```

Comment:
There goes my "first f-strings merged" achievement :(

Reply:
This actually was shrapnel from something else that didn't work, need to remove before merging so your crown is still intact!